web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/ViewsBasicManager.php

```diff
@@ -122,11 +142,20 @@ public function getViewModeLabel($type, $view_mode) {
     return self::ALLOWED_ENTITIES[$type]['view_modes'][$view_mode];
   }
 
+  /**
+   * Returns a tag label given a term ID.
+   */
+  public function getTagLabel($tag) {
+    $term = $this->entityTypeManager()->getStorage('taxonomy_term')->load($tag);
+    return $term->name->value;
```

Comment:
$term may be null if the term does not exist. For example, if someone deletes the taxonomy term. Then I think these node forms will white-screen when trying to generate previews.